templates/activity_stream/tile_impressions/view.sql

```diff
@@ -0,0 +1,82 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.activity_stream.tile_impressions`
+AS
+WITH unioned AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod.activity_stream_stable.impression_stats_v1`
+  UNION ALL
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod.activity_stream_stable.impression_stats_v1`
+  WHERE
+    DATE(submission_timestamp) = CURRENT_DATE()
+),
+impression_data AS (
+  SELECT
+    *
+  FROM
+    unioned
+  WHERE
+    -- make sure data is valid/non-empty
+    ARRAY_LENGTH(tiles) >= 1
+)
+SELECT
+  -- truncate timestamp to seconds
+  TIMESTAMP_TRUNC(submission_timestamp, SECOND) AS submission_timestamp,
+  impression_id AS client_id, -- client_id renamed to impression_id in GCP
+  flattened_tiles.id AS tile_id,
+  addon_version,
+  SUM(CASE WHEN loaded IS NOT NULL THEN 1 ELSE 0 END) AS loaded,
```

Comment:
I believe these `SUM(CASE WHEN x IS NOT NULL THEN 1 ELSE 0 END)` aggregations are equivalent to `COUNT(x)`; see https://cloud.google.com/bigquery/docs/reference/standard-sql/aggregate_functions#count

Reply:
I am ok with this change, thanks for pointing out!